pyqg/layered_model.py

```diff
@@ -137,15 +137,15 @@ def __init__(
     def _initialize_stretching_matrix(self):
         """ Set up the stretching matrix """
 
-        self.S = np.zeros((self.nz, self.nz))
+        self.S = np.zeros((self.nz, self.nz)) # 1/m2
```

Comment:
It took me a little bit to realize that `1/m2` is 1/metres squared. Would it be worth having `# 1/meter^2` just to be explcit?

Just a thoguht to make it more readable.  

I think this is a great addition by the way but won't do a review as I am slowly getting back into the library.

Reply:
Yes, good point -- I actually wasn't sure if I was going to commit these comments initially since they were just for my own sanity as I was checking the units of each diagnostic term, but they probably _are_ useful, and it makes sense to give them the right format.

As a side-note, I wish there was some kind of IDE or explicit way to keep track of units statically when programming (in addition to runtime support from libraries like pint).